lib/cuckoo/common/config.py

```diff
@@ -65,15 +65,28 @@ def get(self, section):
 
 def parse_options(options):
     """Parse the analysis options field to a dictionary."""
-    ret = {}
-    for field in options.split(","):
-        if "=" not in field:
-            continue
+    try:
```

Comment:
Could you write some unit tests to verify that this new logic is both backwards compatible as well as working with the new purpose in mind?